packages/now-cli/src/util/dev/server.ts

```diff
@@ -1952,7 +1954,11 @@ export default class DevServer {
       `Running Dev Command ${chalk.cyan.bold(`“${devCommand}”`)}`
     );
 
-    const port = await getPort();
+    let port = await getPort();
+
+    if (this.frameworkSlug === 'redwoodjs') {
+      port = await getRedwoodPort(cwd);
+    }
```

Comment:
Maybe avoid invoking `getPort()` if we're not going to use the value?

```suggestion
    const port = await (this.frameworkSlug === 'redwoodjs' ? getRedwoodPort(cwd) : getPort());
```